Backend/app/rider-platform/rider-app/Main/src/Beckn/ACL/Common.hs

```diff
@@ -132,6 +132,7 @@ parseBookingDetails order msgId = do
   otp <- start.stopAuthorization >>= (.authorizationToken) & fromMaybeM (InvalidRequest "authorization_token is not present in RideAssigned Event.")
   driverName <- order.orderFulfillments >>= listToMaybe >>= (.fulfillmentAgent) >>= (.agentPerson) >>= (.personName) & fromMaybeM (InvalidRequest "driverName is not present in RideAssigned Event.")
   driverMobileNumber <- order.orderFulfillments >>= listToMaybe >>= (.fulfillmentAgent) >>= (.agentContact) >>= (.contactPhone) & fromMaybeM (InvalidRequest "driverMobileNumber is not present in RideAssigned Event.")
+  driverAlternateNumber <- order.orderFulfillments >>= listToMaybe >>= (.fulfillmentAgent) >>= (.agentContact) >>= (.contactAlternatePhone) & fromMaybeM (InvalidRequest "driverAlternateNumber is not present in RideAssigned Event.")
```

Comment:
Please ensure if we should throw error or not before just adding error. Don't you think it will break whole booking flow for all drivers which doesn't have alternate number, similary backward compatibility.